exercises/zebra-puzzle/instructions.md

```diff
@@ -12,20 +12,20 @@ The following 15 statements are all known to be true:
 1. There are five houses.
 2. The Englishman lives in the red house.
 3. The Spaniard owns the dog.
-4. Coffee is drunk in the green house.
+4. The person in the green house drinks coffee.
 5. The Ukrainian drinks tea.
 6. The green house is immediately to the right of the ivory house.
-7. The Old Gold smoker owns snails.
-8. Kools are smoked in the yellow house.
-9. Milk is drunk in the middle house.
+7. The snail owner likes to go dancing.
+8. The person in the yellow house is a painter.
+9. The person in the middle house drinks milk.
 10. The Norwegian lives in the first house.
-11. The man who smokes Chesterfields lives in the house next to the man with the fox.
-12. Kools are smoked in the house next to the house where the horse is kept.
-13. The Lucky Strike smoker drinks orange juice.
-14. The Japanese smokes Parliaments.
+11. The person who enjoys reading lives in the house next to the person with the fox.
```

Comment:
"The person who enjoys reading lives"

Oooh, we've introduced a seer! 😁

Reply:
The fox house is haunted?

Reply:
I don't think confusing is a goal here. Also, this already got merged.